index.html

```diff
@@ -83,37 +84,181 @@ <h5 class="card__title"><img src="./img/svg/suitcase.svg" class="card-icon"
+                        <span data-rate="2"></span>
+                        <span data-rate="3"></span>
+                        <span data-rate="4"></span>
+                        <div class="empty-stars">
+                            <span data-rate="5"></span>
+                        </div>
+                    </div>
+                    <h6 class="card__text">Slate helps you see how many more days you need to work to reach your
+                        financial goal.
+                    </h6>
+                    <div class="client-info">
+                        <img src="./img/svg/client-vivienne-roche.svg" alt="Vivienne Roche photo" class="client__photo">
+                        <div>
+                            <a href="#!" class="client__name">Vivienne Roche</a>
+                            <h6 class="client__job">CEO</h6>
+                        </div>
+                    </div>
+                </div>
+
+                <div class="grid-card" id="card-items">
+                    <div class="stars">
+                        <span data-rate="1"></span>
+                        <span data-rate="2"></span>
+                        <span data-rate="3"></span>
+                        <span data-rate="4"></span>
+                        <div class="empty-stars">
+                            <span data-rate="5"></span>
+                        </div>
+                    </div>
+                    <h6 class="card__text">Slate helps you see how many more days you need to work to reach your
+                        financial goal.
+                    </h6>
+                    <div class="client-info">
+                        <img src="./img/svg/client-roger-brown.svg" alt="Roger Brown photo" class="client__photo">
+                        <div>
+                            <a href="#!" class="client__name">Roger Brown</a>
+                            <h6 class="client__job">Project Manager</h6>
+                        </div>
+                    </div>
+                </div>   
+            </div>
     </section>
 
-</body>
+    <section class="our-team">
+        <div class="container">
+            <div class="our-team__content">
+                <h2 class="content__general-title">Meet Our Team</h2>
+                <p class="content__text">Problems trying to resolve the conflict between <br> the two major realms of
+                    Classical physics: Newtonian mechanics<p>
+            </div>
 
+            <div class="grid-container">
+                <div class="grid-card" id="card-staff">
+                    <div class="staff-info">
+                        <img src="./img/staff-ashley-fletcher.png" alt="Ashley Fletcher photo" class="staff__photo">
+                        <div>
+                            <div class="staff__content">
+                                <h5 class="staff__name">Ashley Fletcher</h5>
+                                <h6 class="staff__job">CEO</h6>
```

Comment:
You can style capital letters in css.

Reply:
Oh, yees, exactly, thank you! I forgot about the possibility - "text-transform: uppercase;"